tests/document.js

```diff
@@ -0,0 +1,212 @@
+          });
+    });
+
+    it('gets documents with an offset and a limit', function (done) {
+      request
+          .get('/api/documents')
+          .set('x-access-token', token)
+          .set('limit', 2)
+          .set('skip', 1)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(2);
+            expect(res.body[0].title).to.equal('User');
+            done();
+          });
+    });
+
+    it('gets number of documents specified published on a certain date', function (done) {
+      var today = new Date();
+      var day = today.getDate();
+      var month = today.getMonth() + 1;
+      var year = today.getFullYear();
+      var date =  year + '-0' + month + '-' + day;
+
+      request
+          .get('/api/documents?date=' + date)
+          .set('x-access-token', token)
+          .set('limit', 2)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(2);
+            done();
+          });
+    });
+
+    it('returns message if no documents are found', function (done) {
+      request
+          .get('/api/documents?date=2016-09-13')
+          .set('x-access-token', token)
+          .expect({message: 'No documents for this date'}, done);
+    });
+
+    it('gets documents published by the same role level(admin)', function (done) {
+      request
+          .get('/api/documents?role=admin')
+          .set('x-access-token', token)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(4);
+            done();
+          });
+    });
+
+    it('gets documents published by the same role level(user)', function (done) {
+      request
+          .get('/api/documents?role=user')
+          .set('x-access-token', token)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(3);
+            done();
+          });
+    });
+  });
+
+  describe('/documents/:id test suite', function () {
+    it('returns specific document', function (done) {
+      request
+        .get('/api/documents/' + documentId)
+        .set('x-access-token', token)
+        .end(function (err, res) {
+          expect(res.status).to.equal(200);
+          expect(res.body).to.exist;
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.keys('_id', 'title', 'content', 'role', 'modifiedAt', 'ownerId', 'accessLevel', 'createdAt', '__v');
```

Comment:
This could be any document. Confirm with a unique value such as the value of the title field.